src/__tests__/amountQuotePlugin.test.ts

```diff
@@ -28,17 +28,17 @@ describe('creditCardPlugin', function () {
     test('overLimit', function () {
       const errors: FiatProviderError[] = [new FiatProviderError({ providerId: '', errorType: 'overLimit', errorAmount: 50 })]
```

Comment:
This should have been committed in the previous commit. But I still suggest reverting the string renames anyway